components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -3131,7 +3131,7 @@ class CGGetPerInterfaceObject(CGAbstractMethod):
     constructor object).
     """
     def __init__(self, descriptor, name, idPrefix="", pub=False):
-        args = [Argument('*mut JSContext', 'cx'),
+        args = [Argument('&mut SafeJSContext', 'cx'),
```

Comment:
Why do we need a reference at all instead of `Argument('SafeJSContext')`?

Reply:
To not transfer ownership to the function? Its either that or deriving `Copy` for SafeJSContext i believe.

Reply:
Deriving Copy is something we want.

Reply:
Ok i'll update the code :+1: